docs/src/components/Nav.astro

```diff
@@ -32,28 +33,17 @@ import SocialHandlesIcons from "./SocialHandlesIcons.astro";
     </div>
     <div class="navbar-menu">
       <div class="navbar-start"></div>
-      <div class="navbar-end">
-        <a
-          class={`navbar-item`}
-          href="https://playground.dicedb.io"
-          target="_blank"
-        >
-          <span class="has-text-weight-bold"> Playground</span>
-        </a>
+      <div class="navbar-end" style="gap:0.3rem;">
```

Comment:
better not to use inline style definitions like this. Please keep styling consistent with existing practices.

Reply:
For gap, I could find 2 options.

- Adding `gap:0.3rem` on `.navbar-end` in `main.css`. 
- Adding `mr-1` (for desktop), and `mb-1` (for <1024px) on each link in navbar.

I prefer the first one, so pushed that. Let me know if you can think of another approach to better handle this